.travis.yml

```diff
@@ -16,5 +16,6 @@ matrix:
         - docker run --rm -i hadolint/hadolint < Dockerfile
     - env:
         - task=Docker build
+      if: branch == master && (type != pull_request)
```

Comment:
This will prevent MaxMinx license key leaking via pull request, but prevent pull request building tests.